pep-3146.txt

```diff
@@ -134,7 +134,7 @@ integer-optimized ``a + b`` snippet receives two strings, we must support this.
 We cannot change Python semantics. Each of these sections of optimized machine
 code is preceded by a `guard`, which checks whether the simplifying assumptions
 we made when optimizing still hold. If the assumptions are still valid, we run
-the optimized machine code; if they are not, we revert back to the interpreter
```

Comment:
Not sure I agree with this one. "revert back" may be redundant, but "revert" suggests a git revert to me. Using "return" would avoid the problem.

Reply:
Agreed, I'd rather undo (ahem, revert) this change. "return" also has a precise technical meaning, I'd rather not make changes that may subtly change the meaning of the text.